README.md

```diff
@@ -1,2 +1,3 @@
 # GenesisCommander
 Genesis Commander is a full featured file manager for Haiku
+![GenesisCommander](genesiscommander.png)
```

Comment:
Can you add a "." at the end of the sentence, and an empty line above the screenshot, please?
Also, the heading should be "Genesis Commander".